db/migrate/20171006140658_modify_feedbacks_to_be_polymorphic.rb

```diff
@@ -0,0 +1,17 @@
+class ModifyFeedbacksToBePolymorphic < ActiveRecord::Migration[5.1]
+  def up
+    add_reference :feedbacks, :feedbackable, polymorphic: true, index: true
+    Feedback.with_deleted.each do |feedback|
+      feedback.update(feedbackable_id: feedback.assignment_id, feedbackable_type: 'Assignment')
```

Comment:
`feedback.update(feedbackable: feedback.assignment)` should do the same (just FYI, you can leave it as it is)